src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,136 @@
+.contacts {
+  @include page-grid;
+
+  margin-bottom: 64px;
+
+  @include on-tablet {
+    margin-bottom: 120px;
+  }
+
+  @include on-desktop {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 1 / -1;
+    }
+
+    @include on-desktop {
+      grid-column: 1 / -1;
+    }
+  }
+
+  &__content {
+    @include page-grid;
+
+    grid-column: 1 / -1;
+  }
+
+  &__form {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 1 / -1;
+    }
+
+    @include on-desktop {
+      grid-column: 1 / 6;
+    }
+  }
+
+  &__info {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 1 / 4;
+    }
+
+    @include on-desktop {
+      grid-column: 7 / 12;
+    }
+  }
+  &__input {
+    width: 100%;
+    height: 48px;
+    border-radius: 8px;
+    background-color: #f8f8fa;
+    outline: none;
+    border: none;
+    margin-bottom: 16px;
+    color: #7e7e83;
+
+    font-family: 'Space Mono', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 19px;
+
+    &:hover {
+      color: #292929;
+      border: 1px solid#D5D7DE;
+    }
+
+    &:focus {
+      color: #292929;
+      border: 1px solid#292929;
+    }
+
+    &:target {
+      color: #292929;
+    }
+  }
+
+  &__input--text {
+    height: 148px;
+  }
+
+  &__button {
+    font-family: 'Space Grotesk', sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 27px;
+
+    width: 100%;
+    height: 56px;
+    border-radius: 8px;
+    border: 2px solid #f0f1f4;
+
+    margin-bottom: 48px;
+
+    background-color: #f0f1f4;
+
+    &:hover {
+      color: #292929;
+
+      background-color: #eeeeee3b;
+      border: 2px solid #f0f1f4;
+    }
+
+    &:active {
+      color: #fff;
+      background-color: #111;
+    }
+  }
+
+  &__phone {
+    color: #9393a3;
+  }
+
+  &__name {
+    color: #9393a3;
+  }
+
+  &__email {
+    color: #9393a3;
+  }
+
+  &__text {
+    font-family: 'Space Mono', sans-serif;
```

Comment:
You can move all fonts in variables. 
And don't forget about font inheriting.

Reply:
Moved all fonts in variables. And don't forget about font inheriting))))